src/styles/header.scss

```diff
@@ -0,0 +1,71 @@
+$color-acdc: #00acdc;
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-moyo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
```

Comment:
Move this styles to logo-moyo.scss file
You can't set width and height for BEM block